<launch>
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />
  <include file="$(find mongodb_store)/launch/mongodb_store.launch">
    <arg name="db_path" value="/home/jlurz/mongodb"/>
  </include>

  <include file="$(find pr2_moveit_config)/launch/move_group.launch" />

  <node pkg="kinematics_cache" type="orientation_updater" name="left_arm_orientation_updater" output="screen">
    <param name="arm" value="left_arm" />
  </node>
  <!-- <node pkg="kinematics_cache" type="orientation_updater" name="right_arm_orientation_updater" output="screen">
    <param name="arm" value="right_arm" />
  </node> -->
</launch>
